release-notes/README.md

```diff
@@ -4,6 +4,28 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## Aug 6, 2021
+
+### Adobe Experience Platform Mobile SDK plugins for React Native
+
+This major release introduces breaking changes that will bring:
```

Comment:
This major release introduces support for the following:

- Support for React Native version 0.60.+
- Autolinking for native dependencies and removal of the bundled SDK binaries (xcframework) from the React Native module.
- Dynamic versions for native dependencies to always load the latest SDK.
- Removal of several deprecated APIs.

Please note that this release introduces breaking changes. For more details, please see the release notes of the following plugins: